common/components/common/projects/AboutProjectDisplay.jsx

```diff
@@ -222,7 +222,11 @@ class AboutProjectDisplay extends React.PureComponent<Props, State> {
 
           <div className='AboutProjects-details'>
             <div id='project-details'>
-              {project.project_description}
+              {
+                project.project_description + "\n" +
```

Comment:
I think it makes more sense to have a double space between paragraphs.  Our existing projects tend to use double spaces when they're typing up their descriptions.